src/doc/build-script.md

```diff
@@ -209,17 +209,21 @@ some code. Let's see what's inside the build script:
 ```rust,no_run
 // build.rs
 
-use std::os;
-use std::io::File;
+use std::env;
+use std::fs::File;
+use std::io::Write;
+use std::path::Path;
 
 fn main() {
-    let dst = Path::new(os::getenv("OUT_DIR").unwrap());
+    let dst = env::var_os("OUT_DIR").unwrap();
+    let dst = Path::new(&dst);
     let mut f = File::create(&dst.join("hello.rs")).unwrap();
-    f.write_str("
+
+    f.write_all("
         pub fn message() -> &'static str {
             \"Hello, World!\"
         }
-    ").unwrap();
+    ".as_bytes()).unwrap();
```

Comment:
Instead of using `.as_bytes()` this could use the byte-string (`b"..."`) syntax as well.